maxim/tests/test_llamaindex.py

```
@@ -0,0 +1,128 @@
+class TestLlamaIndex(unittest.TestCase):
+    def setUp(self):
+        """Set up test fixtures before each test method."""
+        # This is a hack to ensure that the Maxim instance is not cached
+        if hasattr(Maxim, "_instance"):
+            delattr(Maxim, "_instance")
+
+        self.maxim = Maxim({"base_url": baseUrl})
+        self.logger = self.maxim.logger()
+        instrument_llamaindex(self.logger)
+
+    def test_llamaindex_complex_agent(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_llamaindex_complex_agent`

Add return type annotation: `None`

(ANN201)